src/authenticationService/index.js

```diff
@@ -0,0 +1,13 @@
+import * as firebase from 'firebase'
+
+const getCurrentUserUID = () => {
+  try {
+    return firebase.auth().currentUser.uid
+  } catch (error) {
+    throw new Error('Authentication problem: Not logged in.')
```

Comment:
Instead of determining the error type by the message of it, we should have different types of error: `AuthenticationError`, `NetworkError`, etc, so that we can react to them properly.